DOCUMENTATION.md

```diff
@@ -441,6 +441,66 @@ personalisation={
 </details>
 
 
+### Send a precompiled Letter
+
+This is an invitation-only feature. Contact the GOV.UK Notify team on the [support page](https://www.notifications.service.gov.uk/support) or through the [Slack channel](https://ukgovernmentdigital.slack.com/messages/govuk-notify) for more information.
+
+#### Method
+
+```javascript
+var response = notifyClient.sendPrecompiledLetter(
+    reference,      // Reference to identify the notification
+    pdf_file        // PDF File object
+)
+```
+
+#### Arguments
+
+##### `reference` (required)
+
+A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. It must not contain any personal information such as name or postal address.
```

Comment:
First sentence doesn't quite make sense, as this reference doesn't have to be unique. How do you feel about something like

> This reference can identify a single unique notification or a batch of notifications

Reply:
Ahh, that sounds better indeed, I will adjust it ☺️